client/src/App.js

```diff
@@ -3,27 +3,31 @@ import FloatingActionButton from './components/FloatingActionButton'
 import SearchBooks from './components/SearchBooks'
 import styled from 'styled-components/macro'
 import useBookData from './hooks/useBookData'
+import BookDetail from './components/BookDetail'
```

Comment:
You could set a fixed order in which you import your content into your components. Then the files are a bit clearer overall. (e.g. always start with dependencies and end with components)